client-app/src/components/PlayerCount.tsx

```diff
@@ -0,0 +1,51 @@
+import getAzureFunctions from 'getAzureFunctions';
+import React, { useCallback, useEffect, useState } from 'react'
+import styles from "./PlayerCount.module.scss";
+
+interface PlayerCountProps {
+    courseId: number
+    className: any
+}
+
+export const PlayerCount: React.FC<PlayerCountProps> = (props) => {
+    const [numberOfPlayers, setNumberOfPlayers] = useState(0);
+    const EVENTS_INTERVAL = 2500;
+    const getNumberOfPlayers = useCallback(
+        () => {
+            const url = new URL(getAzureFunctions().GetNumberOfPlayers);
+            url.searchParams.append("course_id", props.courseId.toString());
+            fetch(url.toString())
+                .then(response => response.json())
+                .then(json => {
+                    setNumberOfPlayers(json.NumberOfPlayers);
+                })
+                .catch(err => {
+                    console.log(err);
+                });
+        },
+        [setNumberOfPlayers, props.courseId]
+    );
+
+    useEffect(
+        () => {
+            // Immediately fetch number of players
+            //getNumberOfPlayers();
+            // Continue to fetch stage events + number of players every EVENTS_INTERVAL milliseconds
+            console.log("starting ping interval for number of players!");
+            //const intervalNumberOfPlayers = setInterval(getNumberOfPlayers, EVENTS_INTERVAL);
```

Comment:
Why is this commented out? Otherwise it'll never fetch the number of players and always show "Player Count: 0"

Reply:
Forgot to remove that. I did that so that it wouldn't bombard the server while styling.

Reply:
Fixed